src/createContext.ts

```diff
@@ -153,7 +153,8 @@ const createNativeStorage = (): NativeStorage => ({
   gpu: new Map(Object.entries(gpu_lib)),
   maxExecTime: JSSLANG_PROPERTIES.maxExecTime,
   evaller: null,
-  loadedModules: {}
+  loadedModules: {},
+  loadedModulesTypes: {}
```

Comment:
idk if it's my English but I think it would be `loadedModuleTypes` instead of `loadedModulesTypes`? What do you think?